JPF/bad_buildings.sql

```diff
@@ -71,7 +105,7 @@ left join
 on
 	c.complaintid = cp.complaintid
```

Comment:
What is "c"? If it's hpd_complaints above, please add "AS" as done elsewhere in the statement. As a SQL newbie, this took me a while to figure out.
Also consider using clearer variable / alias names, especially given some aliases are used before the alias is even defined.